AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DisplayMapperAdded.py

```diff
@@ -129,52 +188,233 @@ def AtomEditorComponents_DisplayMapper_AddedToEntity():
+            # Type property cannot be set currently. as a workaround we are calling an ebus to set the operation type
+            # display_mapper_component.set_component_property_value(
+            #     AtomComponentProperties.display_mapper('Type'), DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            render.DisplayMapperComponentRequestBus(
+                bus.Broadcast, "SetDisplayMapperOperationType", DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            general.idle_wait_frames(3)
+            set_type = render.DisplayMapperComponentRequestBus(bus.Broadcast, "GetDisplayMapperOperationType")
+            Report.info(f"DiplayMapperOperationType: {set_type}")
+
+            # 7. Set Enable LDR color grading LUT property True
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Enable LDR color grading LUT'), True)
+            Report.result(
+                Tests.enable_ldr_color_grading_lut,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Enable LDR color grading LUT')) is True)
+
+
+            # 8. Set Override Defaults to False
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Override Defaults'), False)
+            Report.result(
+                Tests.override_defaults,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Override Defaults')) is False)
+            # Set Override Defaults to True
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Override Defaults'), True)
+            Report.result(
+                Tests.override_defaults,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Override Defaults')) is True)
+
+            # 9. Set Alter Surround to True
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Alter Surround'), True)
+            Report.result(
+                Tests.alter_surround,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Alter Surround')) is True)
+            # Set Alter Surround to False
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Alter Surround'), False)
+            Report.result(
+                Tests.alter_surround,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Alter Surround')) is False)
+
+            # 10. Set Alter Desaturation to True
```

Comment:
Same here and for all the other "True/False" steps - it should match the docstring and then an extra comment for each specific sub-step within that main step

Reply:
updated